src/component/spinner.js

```diff
@@ -62,11 +63,55 @@ export const LoadNetworkSpinner = ({ percentage, msg, style }) => (
 );
 
 LoadNetworkSpinner.propTypes = {
+  continuous: PropTypes.bool,
   percentage: PropTypes.number.isRequired,
   msg: PropTypes.string.isRequired,
   style: ViewPropTypes.style,
 };
 
+//
+// ContinuousLoadNetworkSpinner
+//
+
+export class ContinuousLoadNetworkSpinner extends Component {
+  constructor(props) {
+    super(props);
+    this.increasePercentage = this.increasePercentage.bind(this);
+    this.state = {
+      percentage: 0,
+    };
+  }
+  componentDidMount() {
+    this.intervalId = setInterval(this.increasePercentage, 10);
+  }
+  render() {
+    const { msg, style } = this.props;
+    const { percentage } = this.state;
+    return (
+      <LoadNetworkSpinner
+        msg={msg}
+        percentage={percentage}
+        style={style}
+        continuous={true}
+      />
+    );
+  }
+  increasePercentage() {
+    const { percentage } = this.state;
+    this.setState({
+      percentage: (percentage + 0.01) % 1,
+    });
+  }
```

Comment:
Sorry if this didn't come across in the design. But the percentage should stay 25% statically for this spinner as it's just a generic wait screen that is used without any progress.

Reply:
okay cool. i noticed the startup spinner didn't look quite right but hadn't dug in to see what was going on.

Reply:
do you mean that this line should be `25` instead of `store.percentSynced`? https://github.com/lightninglabs/lightning-app/blob/master/src/view/loader-syncing.js#L24

Reply:
My mistake that as the `LoaderView`. My point was that you should be able just set `percentage={0.25}` on the `LoadNetworkSpinner` for the `ContinuousLoadNetworkSpinner` and get the desired result.

Reply:
I'm still not sure what exactly needs to be done to clear up this thread of comments. I'll move the two arc generation functions into a helper file, and write a couple tests. But if there is something else you need RE: this thread, either be overly specific so I know what to do, or maybe just wait til I'm done and add your own commit

Reply:
Ok thanks @willpiers. Sounds good. Let me know when you're ready with your changes.